src/scheduler/base.rs

```diff
@@ -178,6 +180,7 @@ impl Worker {
     /// config_path: path to the config file we are working with
     #[instrument]
     fn new(worker_type: WorkerType, survey_name: Survey, config_path: String) -> Worker {
+        let id = Uuid::new_v4();
```

Comment:
wait is this reintroducing a uuid-based worker id?

Reply:
Yeah, there needs to be a way to distinguish workers in Prometheus. I was started by stringifying the `ThreadId` value, but that resulted in strings like "ThreadId(7)" and I thought UUID would be better. I was kind of plus or minus on making the UUID a field of the `Worker` struct: we don't have a use for it now, but I'm not sure we won't ever need it, so I named it with a leading underscore. I'm happy to remove the field if you don't want it.